components/gly_gen/glycosight/sup_functions.ts

```diff
@@ -2,7 +2,7 @@ import { GlycoSightOutputType, GlycoSightFileURLType } from "./data_models";
 import { GlycoSightOutputNode } from ".";
 import { z } from "zod";
 
-const devGSURL = "http://localhost:5000/api/"
+const devGSURL = process.env.NODE_ENV === "development" ? "http://localhost:5000/api/" : "https://aws.glygen.org/glycosight/api/"
```

Comment:
The main issue I see with this is that other people who don't have your API running locally can't test your node locally, potentially in a pipeline to do something else. I can update it..

Reply:
Right, doing local development doesn't allow DNS resolution of the `drs` server when pushing information to AWS. (`localhost` is telling `aws.glygen.org/glycosight` to retrieve files from the `drs` location at `localhost/gh4ga...`, which will be problematic for the `aws.glygen.org` server.) 

I didn't think of a clever workaround, but other folks would certainly be able to stand up local instances of our API.

If you have a workaround in mind, I'm certainly open to suggestions!

Reply:
I think the workaround here may be to keep cached results for the example file within the glycosight package. When other people are working with our API, that will give a valid result MetaNode to continue development and will be available regardless of where development is taking place.